### PR TITLE
categories.js in model got added

### PR DESCRIPTION
**Pull request by thakurdeepak**
@aadarshp31  please check it once

### CODE REVIEW

File: backend/models/categories.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            maxlength: 25,
+            unique: true,
+            trim: true,
+        },
+        {
+            timestamps : true
+        }
+    );
+
+    module.exports = mongoose.model("Categories",categoriesSchema);
```
Comment (aadarshp31) [inline backend/models/categories.js:17]: The name of the model must be "Category"

File: backend/models/categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const categoriesSchema = new mongoose.Schema(
+    {
```
Comment (aadarshp31) [inline backend/models/categories.js:4]: Unnecessary "{" sign, please remove it

File: backend/models/categories.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
```
Comment (aadarshp31) [inline backend/models/categories.js:1]: ';' is not required in javascript. Please do enable format on save feature in vscode and remove ';' signs

File: backend/models/categories.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+const categoriesSchema = new mongoose.Schema(
+    {
+        name: {
+            type: String,
+            required: true,
+            maxlength: 25,
+            unique: true,
+            trim: true,
+        },
```
Comment (aadarshp31) [inline backend/models/categories.js:11]: This "{" needs to be closed after line number 11

File: backend/package.json [hunk 5]
```diff
@@ -30,6 +30,7 @@
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.11.11",
     "nodemon": "^2.0.7",
+    "save": "^2.4.0",
```
Comment (aadarshp31) [inline backend/package.json:33]: You do not need to install a package called "save" for this feature. Please uninstall this package using the command "npm uninstall save"

**Review by cdacgroup16 — APPROVED** [review]

**Review by aadarshp31 — APPROVED** [review]

**Pull request merged**